src/js/components/modals/JobFormModal.js

```diff
@@ -54,6 +55,16 @@ const responseAttributePathToFieldIdMap = {
   '/labels': 'labels'
 };
 
+let cleanJSONspec = function (jsonDefinition) {
```

Comment:
If this is similar to [the one in ServiceFormModal.js](https://github.com/dcos/dcos-ui/blob/master/src/js/components/modals/ServiceFormModal.js#L109) shouldn't this be moved to the utility functions?